src/main/java/Duke.java

```diff
@@ -6,5 +9,97 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("\t------------------------------------------");
+        System.out.println("\tHello there! I'm Duke.");
+        System.out.println("\tWhat can I help you with?");
+        System.out.println("\t__________________________________________\n");
+
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        int taskCount = 0;
```

Comment:
Instead of using the `taskCount` variable, is it possible to use `tasks.length` instead, so there is one less variable to manage?

Reply:
Good idea. Fixed it already!